pilot/pkg/serviceregistry/kube/controller/controller.go

```diff
@@ -358,6 +326,14 @@ func (c *Controller) GetService(hostname host.Name) (*model.Service, error) {
 	return c.servicesMap[hostname], nil
 }
 
+func GetLocalityFromTopology(topology map[string]string) string {
```

Comment:
does this need to be public? Seems like it should just be private and co-located with the endpointslice controller?

Reply:
made private